app/views/admin/dashboard/index.html.erb

```diff
@@ -24,6 +24,14 @@
     <h3>Product development</h3>
     <ul>
       <li><a href="https://insidegovuk.blog.gov.uk/">Inside GOV.UK blog</a></li>
+      <li><%= link_to(t('whats_new.dashboard_link_label'),
```

Comment:
The dashboard stuff shouldn't be in the same locale file as the rest of the stuff (since it should match views). Can we just hard code this?

Reply:
Yep - can be hard coded, will move

Reply:
done